concent_api/api_testing_common.py

```diff
@@ -0,0 +1,187 @@
+import datetime
+import http.client
+import json
+import requests
+import sys
+
+from collections                    import namedtuple
+
+from golem_messages.exceptions      import MessageError
+from golem_messages.message         import ComputeTaskDef
+from golem_messages.message         import Message
+from golem_messages.message         import TaskToCompute
+from golem_messages.shortcuts       import dump
+from golem_messages.shortcuts       import load
+
+
+ProtocolConstants = namedtuple(
```

Comment:
This could be [typing.NamedTuple](https://docs.python.org/3/library/typing.html#typing.NamedTuple).

Sorry for not pointing it out before but only recetly realized that there are typed named tuples in Python.

Reply:
done